colearn_pytorch/test_pytorch_learner.py

```diff
@@ -26,8 +27,8 @@
 from colearn_pytorch.pytorch_learner import PytorchLearner
 
 # torch does not correctly type-hint its tensor class so pylint fails
-MODEL_PARAMETERS = [torch.tensor([3, 3]), torch.tensor([4, 4])]  # pylint: disable=not-callable
-MODEL_PARAMETERS2 = [torch.tensor([5, 5]), torch.tensor([6, 6])]  # pylint: disable=not-callable
+MODEL_PARAMETERS = OrderedDict({'param1': torch.tensor([3, 3]), 'param2': torch.tensor([4, 4])})  # pylint: disable=not-callable
```

Comment:
why we need this? does this mean we have to modify model parameters structure when we use keras learner?

Reply:
In Keras there is a `model.get_weight()`, which returns with all the weights (the trainable and the non-trainable too) as a list of numpy arrays.

In PyTorch there is a `model.parameters()` function, which returns all the trainable weights (and does not return the non-trainable at all, [see](https://discuss.pytorch.org/t/difference-between-state-dict-and-parameters/37531)) as a list of torch tensors.

Also in PyTorch there is a `model.state_dict()` function, which returns all the parameters (both trainable and non-trainable) as an OrderedDict. 

In the new update I am transitioning from `model.parameters()` to `model.state_dict()` to share the non-trainable parameters too (batch norm running mean, var for example). I changed the pytorch tests to use the state_dict() format. It does not affect the Keras learner.

Reply:
so using state_dict is optional and it is still backward compatible?

Reply:
both the model proposal and the model acceptance was modified. this way the demos are running without any modification, so it is backward compatible in that sense. (the exact model format is changed, that cannot be backward compatible, as we are saving/loading more information than previously, but the API part is unchanged)